docs/getting-started/install/kubernetes.rst

```diff
@@ -33,25 +33,17 @@ although you will need to make modifications to the deployment manifests.
    # Create a namespace to run cert-manager in
    kubectl create namespace cert-manager
 
-As part of the installation, cert-manager also deploys a
-`ValidatingWebhookConfiguration`_ resource in order to validate that the
-Issuer, ClusterIssuer and Certificate resources we will create after
-installation are valid.
-
-In order to deploy the ValidatingWebhookConfiguration, cert-manager creates
-a number of 'internal' Issuer and Certificate resources in its own namespace.
-
-This creates a chicken-and-egg problem, where cert-manager requires the
-webhook in order to create the resources, and the webhook requires cert-manager
-in order to run.
-
-We avoid this problem by disabling resource validation on the namespace that
-cert-manager runs in:
-
-.. code-block:: shell
-
-   # Disable resource validation on the cert-manager namespace
-   kubectl label namespace cert-manager cert-manager.io/disable-validation=true
+As part of the installation, cert-manager also deploys a webhook deployment as
+an `APIService`_. This can cause issues when uninstalling cert-manager if the
+API service still exists but the webhook is no longer running as the API server
+is unable to reach the validating webhook. Ensure to follow the documentation
+when uninstalling cert-manager. TODO (@joshvanl): add uninstalling documentation
+link when this has been created.
+
+The webhook enables cert-manager to implement validation and mutating webhooks
+on cert-manager resources. A `ValidatingWebhookConfiguration`_ resource is
+deployed to validate cert-manager resources we will create after installation.
+No mutating webhooks are currently implemented.
```

Comment:
We do create a MutatingWebhookConfiguration resource fwiw, but don't actually mutate anything with it